pkgs/by-name/en/ente-auth/package.nix

```
@@ -78,7 +78,7 @@ flutter324.buildFlutterApplication rec {
   ];
 
   postInstall = ''
-    FAV=$out/app/data/flutter_assets/assets/icons/auth-icon.png
+    FAV=$out/app/ente-auth/data/flutter_assets/assets/icons/auth-icon.png
     ICO=$out/share/icons
 
     install -D $FAV $ICO/ente-auth.png
```

```suggestion
    install -D $out/app/ente-auth/data/flutter_assets/assets/icons/auth-icon.png $out/share/pixmaps/ente-auth.png
```

Delete the below lines. These should be useless

---

`ICO` and `FAV` are also used in a for loop right below, I think its cleaner to keep those variables.

---

I'm saying the following should be removed. share/pixmaps is stipulated by the freedesktop specification, and this path does not need to specify the size. I don't think zooming with imagemagick is a good way.

Also remove `imagemagick` from `nativeBuildInputs` and function formal.

---

Thanks for explaining this! Even though `pixmaps` is deprecated, it seems like better choice than creating images with `imagemagick`.

---

Let's move this to a follow-up PR as it's irrelevant to this fix, so we can unbreak the build first.